src/app/api/chat/stream/route.ts

```
@@ -0,0 +1,218 @@
+  try {
+    session = await getServerSession(authConfig);
+  } catch (error) {
+    console.error("Error fetching session:", error);
+    return new NextResponse('Authentication error', { status: 500 });
+  }
+
+  if (!session?.user?.id) {
+    return new NextResponse('Unauthorized', { status: 401 });
+  }
+
+  let reqBody;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)